app/views/sidebar.ts

```diff
@@ -597,20 +597,24 @@ const resultsContent = (
         let orgClass = '';
         let orgName = '';
 
-        if (organizations.provinces[result.event.collection]) {
-            orgClass = 'province';
-            orgName = organizations.provinces[result.event.collection];
-        } else if (organizations.municipalities[result.event.collection]) {
-            orgClass = 'municipality';
-            orgName = 'Gemeente ' + organizations.municipalities[result.event.collection];
+        const match = result.index.match(/(ori|osi)_([a-z-]+)_\d+/);
+
+        if (match && match[1] && match[2]) {

```

Comment:
if there is a match there is no way match[1] and match[2] are not set and in this case filled